### PR TITLE
Add external link to starlette documentation for further details

### PR DESCRIPTION
**Pull request by hellocoldworld**

### CODE REVIEW

File: docs/en/docs/advanced/testing-websockets.md [hunk 1]
```diff
@@ -7,3 +7,6 @@ For this, you use the `TestClient` in a `with` statement, connecting to the WebS
 ```Python hl_lines="27 28 29 30 31"
 {!../../../docs_src/app_testing/tutorial002.py!}
 ```
+
+## More info
```
Comment (Kludex) [inline docs/en/docs/advanced/testing-websockets.md:11]:
```suggestion:-0+1
## More info
```

Comment (Kludex) [inline docs/en/docs/advanced/testing-websockets.md:11]:
```suggestion
```

File: docs/en/docs/advanced/testing-websockets.md [hunk 2]
```diff
@@ -7,3 +7,6 @@ For this, you use the `TestClient` in a `with` statement, connecting to the WebS
...
+For further deatails, check Starlette's documentation for <a href="https://www.starlette.io/testclient/#testing-websocket-sessions" class="external-link" target="_blank">testing WebSockets</a>.
```
Comment (Kludex) [inline docs/en/docs/advanced/testing-websockets.md:12]:
```suggestion
!!! note For further details, check Starlette's documentation for <a href="https://www.starlette.io/testclient/#testing-websocket-sessions" class="external-link" target="_blank">testing WebSockets</a>.
```

**Review by Kludex — COMMENTED** [review]
What do you think adding a note about it?

**Review by Kludex — APPROVED** [review]

**Comment by tiangolo** [comment]
Good idea, thanks @hellocoldworld ! :cake: 

And thanks @Kludex for the help! :coffee:

**Pull request merged**